lineman/app/modules/navbar/navbar_search_items/navbar_search_comment.haml

```diff
@@ -1,2 +1 @@
-.navbar-search-comment
-  {{comment.body}}
\ No newline at end of file
+.navbar-search-comment{btf-markdown: 'result.blurb'}
```

Comment:
Haven't done this all the way out the frontend yet, but can verify that we can handle rendering blurbs by making em markdown, woo!